### PR TITLE
[DOCS-6157] APA m12 updates

### PR DESCRIPTION
**Pull request by oliveralfresco2018**

### CODE REVIEW

File: process-automation/latest/admin/monitor.md [hunk 1]
```diff
@@ -174,3 +174,4 @@ The properties for each audit event are:
 | Event ID | The unique ID of the event. |
 | Process Instance ID | The process instance ID of the process that the event occurred in. |
 | Event Action | The payload of the event in JSON format. Click to view the details. |
+| Triggers | Whenever a trigger is matched the event is added to the audit table. |
```
Comment (pmartinezga) [inline process-automation/latest/admin/monitor.md:177]: You are describing this table in the admin app 
image

That trigger column is not there, so I think you should drop this addition

File: process-automation/latest/model/interfaces.md [hunk 2]
```diff
@@ -43,6 +43,16 @@ Custom end user actions can be created for the Digital Workspace. This enables c
 * A list of one or more [rules](#rules) to describe which nodes the action will be displayed on.
 * The [feature](#features) the action will be displayed on, for example the header or a context menu.
 
+### Auditing
```
Comment (pmartinezga) [inline process-automation/latest/model/interfaces.md:46]: I would rather use an h2 header and move it to the bottom of the page, I don't think it should be inside the user interface modeling section

**Review by pmartinezga — CHANGES_REQUESTED** [review]
Just minor things

File: process-automation/latest/admin/monitor.md (hunk 1 above)
Comment (oliveralfresco2018) [reply process-automation/latest/admin/monitor.md:177]: Done thanks

File: process-automation/latest/model/interfaces.md (hunk 2 above)
Comment (oliveralfresco2018) [reply process-automation/latest/model/interfaces.md]: done cheers

**Review by pmartinezga — DISMISSED** [review]

**Review by rallegre — APPROVED** [review]
ok for me

File: process-automation/latest/model/connectors/aws.md [hunk 3]
```diff
@@ -160,9 +468,9 @@ The input parameters of the Rekognition connector are:
 | Parameter | Type | Description |
```
Comment (gravitonian) [inline process-automation/latest/model/connectors/aws.md:468]: This table does not display properly

File: process-automation/latest/model/connectors/aws.md [hunk 4]
```diff
@@ -160,9 +468,9 @@ The input parameters of the Rekognition connector are:
...
 | --------- | ---- | ----------- |
 | file | File | *Required.* A [variable]({% link process-automation/latest/model/processes/index.md %}#process-variables) of type file to send for analysis. |
-| mediaType | String | *Optional.* The media type of the file to be analyzed, for example `application/octect-stream`. |
+| mediaType | String | *Optional.* The media type of the file to be analyzed, for example ``/octect-stream`. |
```
Comment (gravitonian) [inline process-automation/latest/model/connectors/aws.md:471]: Looks like too many ticks around the /octet-stream

File: process-automation/latest/model/processes/index.md [hunk 5]
```diff
@@ -7,6 +7,10 @@ Processes are the collection of components that are used to build and represent
 There are three concepts associated with modeling processes to understand and differentiate between:
 
 * **Diagrams** are the container that process definitions are modeled in. A process diagram can contain multiple process definitions when [pools]({% link process-automation/latest/model/processes/bpmn.md %}#pools-and-lanes) are used to separate them.
+Process diagrams are colour coded with the following definitions:
```
Comment (gravitonian) [inline process-automation/latest/model/processes/index.md:10]: The following bullet points does not display indented

**Review by gravitonian — CHANGES_REQUESTED** [review]
Some minor changes needed I think

File: process-automation/latest/model/connectors/aws.md (hunk 3 above)
Comment (oliveralfresco2018) [reply process-automation/latest/model/connectors/aws.md:468]: fixed.

File: process-automation/latest/model/connectors/aws.md (hunk 4 above)
Comment (oliveralfresco2018) [reply process-automation/latest/model/connectors/aws.md:471]: fixed.

File: process-automation/latest/model/processes/index.md (hunk 5 above)
Comment (oliveralfresco2018) [reply process-automation/latest/model/processes/index.md]: Fixed cheers

**Review by gravitonian — APPROVED** [review]

**Pull request merged**